nipype/tests/test_utils.py

```diff
@@ -0,0 +1,17 @@
+from nipype import utils
+
+
+def test_pickle(tmp_path):
+    testobj = 'iamateststr'
+    pickle_fname = str(tmp_path / 'testpickle.pklz')
+    utils.filemanip.savepkl(pickle_fname, testobj)
+    outobj = utils.filemanip.loadpkl(pickle_fname)
+    assert outobj == testobj
+
+
+def test_pickle_versioning(tmp_path):
```

Comment:
What about parameterizing this into a single test?

```Python
@pytest.mark.parametrize("save_ver", (True, False))
@pytest.mark.parametrize("load_ver", (True, False))
def test_pickle(tmp_path, save_ver, load_ver):
    testobj = 'iamateststr'
    pickle_fname = str(tmp_path / 'testpickle.pklz')
    utils.filemanip.savepkl(pickle_fname, testobj, versioning=save_ver)
    outobj = utils.filemanip.loadpkl(pickle_fname, versioning=load_ver)
    assert outobj == testobj
```

Reply:
Good idea.